### ISSUE TITLE
Observation notebook does not work

### ISSUE THREAD

**Issue by benedikt-schesch** [rte-france/Grid2Op#522]
## Environment
 - Grid2op version: `a819a777cc2da67270c84f1887f8cb32fec23308`
 - System: `osx`

## Bug description

get started guide 02_Observation does not execute properly due to missing proper default value.

## How to reproduce
Simply run `getting_started/02_Observation.ipynb` you will see that block 4 fails with the following message:

```
---------------------------------------------------------------------------
Grid2OpException                          Traceback (most recent call last)
Cell In[4], line 1
----> 1 env = grid2op.make(test=True)
      2 obs = env.reset() 

File [~/Git/Grid2Op/getting_started/grid2op/MakeEnv/Make.py:350](https://file+.vscode-resource.vscode-cdn.net/Users/benediktschesch/Git/Grid2Op/getting_started/~/Git/Grid2Op/getting_started/grid2op/MakeEnv/Make.py:350), in make(dataset, test, logger, experimental_read_from_local_dir, _add_to_name, _compat_glop_version, **kwargs)
    347         test = True
    349 if dataset is None:
--> 350     raise Grid2OpException("Impossible to create an environment without its name. Please call something like: \n"
    351                            "> env = grid2op.make('l2rpn_case14_sandbox') \nor\n"
    352                            "> env = grid2op.make('rte_case14_realistic')")
    354 accepted_kwargs = ERR_MSG_KWARGS.keys() | {"dataset", "test"}
    355 for el in kwargs:

Grid2OpException: Grid2OpException "Impossible to create an environment without its name. Please call something like: 
> env = grid2op.make('l2rpn_case14_sandbox') 
or
> env = grid2op.make('rte_case14_realistic')"
```

Problem is that the default value is None which is not what is written in the notebook

**Comment by benedikt-schesch** [comment]
I would very highly recommend adding the execution of all the notebooks to the CI/CD pipeline so changes to the API don't break the user guides

**Comment by benedikt-schesch** [comment]
This problem is solved by #524

**Comment by BDonnot** [comment]
> I would very highly recommend adding the execution of all the notebooks to the CI/CD pipeline so changes to the API don't break the user guides

Hello,

You are correct here, this is what used to be done but at some point I had trouble with their execution (though the notebooks worked perfectly, the CI still send weird error messages) and this is why I deactivated it. There is an issue about that #234 that is not on my priority at the moment (and I need to redo some of the notebooks...)
Too many things to do, not enough time to do it apparently...

**Comment by BDonnot** [comment]
notebook fixed in version 1.9.5 (released a few minutes ago)
